File: PPA/DailyFlash/August/21August/Program2.c


#include<stdio.h>

void main() {

	int unit, totalBill;
	float unitCharge;

	printf("Enter the unit: ");
	scanf("%d",&unit);
	
	if(unit>0 && unit<50)
		unitCharge=0.50;
	else if(unit>51 && unit<150)
		unitCharge=0.75;
	else if(unit>151 && unit<250)
		unitCharge=1.20;
	else 
		unitCharge=1.50;

	printf("Total electricity bill= Rs %.2f\n",unitCharge*unit);
}

File: Java/DailyFlash/August/21August/Program2.java

import java.io.*;

class Electricity {

	public static void main(String[] args) throws IOException {

		InputStreamReader isr = new InputStreamReader(System.in);
		BufferedReader br =new BufferedReader(isr);

		int unit;
		float unitCharge;

		System.out.print("Enter the number of unit: ");
		unit = Integer.parseInt(br.readLine());

		if(unit>0 && unit<=50)
			unitCharge=0.50f;
		else if(unit>51 && unit<=150)
			unitCharge=0.75f;
		else if(unit>151 && unit<=250)
			unitCharge=1.20f;
		else
			unitCharge=1.50f;

		System.out.println("Total electricity unit charges = "+unitCharge*unit);
	}
}

